src/core/rubberbandmodel.h

```diff
@@ -114,6 +115,8 @@ class RubberbandModel : public QObject
 
     void setGeometryType( const QgsWkbTypes::GeometryType &geometryType );
 
+    void setDataFromGeometry( const QgsGeometry &geometry );
```

Comment:
add a docstring with a comment saying that rings and multiparts are discarded.

Reply:
Done.